config/initializers/devise.rb

```diff
@@ -22,7 +22,9 @@ def route(scope)
   # confirmation, reset password and unlock tokens in the database.
   # Devise will use the `secret_key_base` as its `secret_key`
   # by default. You can change it below and use your own secret key.
-  # config.secret_key = 'abcdefg'
+if Rails.env.test?
+    config.secret_key = Rails.application.secret_key_base
+end
```

Comment:
Small whitespace tweak
```suggestion
  if Rails.env.test?
    config.secret_key = Rails.application.secret_key_base
  end
```